tools/cmake/toolchain/rdkb.cmake

```diff
@@ -0,0 +1,80 @@
+## General configuration
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR ARMV7 CACHE STRING "System Processor")
+set(CMAKE_EXPORT_NO_PACKAGE_REGISTRY ON)
+
+# Workaround for https://www.cmake.org/Bug/view.php?id=14075
+set(CMAKE_CROSS_COMPILING ON)
+
+# Platform specific
+set(TARGET_PLATFORM "rdkb" CACHE STRING "Target Platform")
+
+# Base RDKB build dir (e.g. /path/to/atom_rdkbos)
+if (NOT DEFINED ENV{PRPLMESH_PLATFORM_BASE_DIR})
+    message(FATAL_ERROR "PLATFORM_BASE_DIR environment variable not defined!")
+endif()
+set(PLATFORM_BASE_DIR "$ENV{PRPLMESH_PLATFORM_BASE_DIR}" CACHE STRING "Platform Base Directory")
+
+# RDKB build name (e.g. puma7-atom)
+if (NOT DEFINED ENV{PRPLMESH_PLATFORM_BUILD_NAME})
+    message(FATAL_ERROR "PRPLMESH_PLATFORM_BUILD_NAME environment variable not defined!")
+endif()
+set(PLATFORM_BUILD_NAME "$ENV{PRPLMESH_PLATFORM_BUILD_NAME}" CACHE STRING "Platform Build Name")
+
+# Platform toolchain name (e.g. x86_64-linux)
+if (NOT DEFINED ENV{PRPLMESH_PLATFORM_TOOLCHAIN})
+    message(FATAL_ERROR "PRPLMESH_PLATFORM_TOOLCHAIN environment variable not defined!")
+endif()
+set(PLATFORM_TOOLCHAIN "$ENV{PRPLMESH_PLATFORM_TOOLCHAIN}" CACHE STRING "Platform Toolchain")
+
+# Platform toolchain prefix (e.g. i686-rdk-linux)
+if (NOT DEFINED ENV{PRPLMESH_PLATFORM_TOOLCHAIN_PREFIX})
+    message(FATAL_ERROR "PLATFORM_TOOLCHAIN_PREFIX environment variable not defined!")
+endif()
+set(PLATFORM_TOOLCHAIN_PREFIX "$ENV{PRPLMESH_PLATFORM_TOOLCHAIN_PREFIX}" CACHE STRING "Platform Toolchain Prefix")
+
+# Platform directories
+set(PLATFORM_BUILD_DIR          ${PLATFORM_BASE_DIR}/build/tmp/sysroots CACHE STRING "Platform Build Directory")
+set(PLATFORM_STAGING_DIR        ${PLATFORM_BUILD_DIR}/${PLATFORM_BUILD_NAME} CACHE STRING "Platform Staging Directory")
+set(PLATFORM_TOOLCHAIN_DIR      ${PLATFORM_BUILD_DIR}/${PLATFORM_TOOLCHAIN}/usr/bin/${PLATFORM_TOOLCHAIN_PREFIX} CACHE STRING "Platform Toolchain Directory")
+set(PLATFORM_INCLUDE_DIR        ${PLATFORM_STAGING_DIR}/usr/include CACHE STRING "Platform Include Directory")
+set(ENV{PKG_CONFIG_PATH}        "${PLATFORM_STAGING_DIR}/usr/lib/pkgconfig")
+
+# Compiler Definitions
+add_definitions(
+    -DBEEROCKS_RDKB
+)
+
+## Library config
+set(BUILD_SHARED_LIBS ON CACHE BOOL "BUILD_SHARED_LIBS" FORCE)
```

Comment:
I recall that @arnout said these are not needed.

Reply:
Will remove that as well